BEIMA.Backend.Test/MongoService/MongoConnectorTest.cs

```diff
@@ -160,5 +160,121 @@ public void ConnectorCreated_CheckIsConnected_TrueReturned()
             var result = mongo.IsConnected();
             Assert.IsTrue(result);
         }
+
+        //DeviceType tests
```

Comment:
I think you might find using regions to be pretty handy here:
https://www.c-sharpcorner.com/article/using-regions-to-improve-code-readability/

Reply:
Oops, I was meaning to use regions there and was planning on going back to change that. Thanks for catching that, I'll fix that and add another region for Device tests.

Reply:
Fixed [bb6c78f](https://github.com/cs481-ekh/s22-beima/pull/134/commits/bb6c78f547b5e677d7518b3bf4db83c584eb42d7)